index.rst

```diff
@@ -92,53 +138,106 @@ The electometer sits in the electronics box and the cable from the photodiode is
+
 Shutter system
 --------------
 In order to both keep light enclosed when not needed and also to keep dust out of the illumination system, there is a shutter for the output of the beam. The beam travels through glass when the shutter is opened. The shutter can only be commanded fully open or fully closed, not at a point in between. When the shutter has reached one limit or another, it will hit a switch which will confirm to software that it has indeed reached the end of travel. The linear actuator is attached to the top of the structure so must be disassembled before the top of the box can be moved. 
 
 The linear actuator that moves the shutter is controlled with the LabJack. More information can be found on that in `tech note 36 <https://tstn-036.lsst.io>`__.
 
+.. figure:: /_static/shutter_system.png
+ :name: shutter_system
+ :target: ../_images/shutter_system.png
+ :alt: shutter_system
+
+ Shutter system contains shutter, linear actuator and two limit switches.
+
+
+.. figure:: /_static/linact.png
+ :name: linact
+ :target: ../_images/linact.png
+ :alt: linact
+ :scale: 15 %
+
+ Linear actuator for shutter system. 
+
 
 Electronics Cabinet
 -------------------
-The electronics cabinet must be turned off before opening. Inside the cabinet there is a thermostat and fan that keeps the electronics cooled. The thermostat is set at 20C (TBC). 
+The electronics cabinet must be turned off before opening. Inside the cabinet there is a thermostat and fan that keeps the electronics cooled. The thermostat is set at 20C. 
+
+The cabinet includes the control computer (loonie) and the windows machine for the monochromator, both of which are embedded SBCs. There are two serial device servers for the electrometer and chiller and the LabJack with custom board. There are 5V, 12V and 24V voltage supplies, all of which plug into an 8-outlet PDU.
 
-The cabinet includes the control computer (loonie) and the windown machine for the monochromator, both of which are embedded SBCs. There are two serial device servers for the electrometer and chiller and the LabJack with custom board. There are 5V, 12V and 24V voltage supplies, all of which plug into an 8-outlet PDU.
+The electronics cabinet is powered by 220Vac from the dome. Additionally, 6 ethernet cables are needed to run into the electronics cabinet for: electrometer serial device server, chiller serial device server, control computer (loonie), monochromator windows machine, LabJack, PDU. 
 
-The electronics cabinet is powered by 220Vac from the dome. Additionally, 5 ethernet cables are needed to run into the electronics cabinet for: electrometer serial device server, chiller serial device server, control computer (loonie), monochromator windows machine, LabJack, PDU. 
+.. figure:: /_static/elec_cabinet.png
+   :name: elec_cabinet
+   :target: ../_images/elec_cabinet.png
+   :alt: elec_cabinet
+
+   View of electronic cabinet when opened. 
```

Comment:
please fix the label with loonie and put the new name.

Reply:
done